closed/src/java.base/share/classes/com/sun/crypto/provider/NativeGaloisCounterMode.java

```diff
@@ -380,9 +440,6 @@ int encryptFinal(byte[] in, int inOfs, int len, byte[] out, int outOfs)
      * @return the number of bytes placed into the <code>out</code> buffer
      */
     int decrypt(byte[] in, int inOfs, int len, byte[] out, int outOfs) {
-        if ((len % blockSize) != 0) {
-            throw new ProviderException("Internal error in input buffering");
-       }
```

Comment:
Why are these lines removed from `decrypt`, but not `encrypt`?

Reply:
I removed those lines from encrypt also.